File: 프로그래머스/unrated/181837. 커피 심부름/커피 심부름.js
function solution(order) {
    let answer = 0;
    
    order.forEach(_order => {
        if (_order.includes('americano') || _order === 'anything') {
            answer += 4500;
        } else {
            answer += 5000;
        }
    });
    
    return answer;
}

File: 프로그래머스/unrated/181837. 커피 심부름/커피 심부름.java
class Solution {
    public int solution(String[] order) {
        int result = 0;
        
        for (String cafe : order) {
            if (cafe.contains("americano")) {
                result += 4500;
            } else if (cafe.contains("cafelatte")) {
                result += 5000;
            } else {
                result += 4500;
            }
        }
        
        return result;
    }
}